libyul/AssemblyStack.cpp

```diff
@@ -226,6 +226,48 @@ MachineAssemblyObject AssemblyStack::assemble(Machine _machine) const
 	return MachineAssemblyObject();
 }
 
+std::pair<MachineAssemblyObject, MachineAssemblyObject> AssemblyStack::assemble(std::string const& _runtimeName) const
```

Comment:
Can't this function be combined with `assemblyAndGuessRuntime`? For example, the name could be a `std::optional`.

Reply:
so you want to keep the heuristic part as well if no name was given?

Reply:
I want to reduce code duplication.